.travis.yml

```diff
@@ -3,7 +3,8 @@ lein: 2.7.1
 script:
   - lein test
   - lein test :generative
+  - lein with-profile dev,recent-clj test
 # Workaround for https://github.com/travis-ci/travis-ci/issues/4691
 sudo: required
 jdk:
-  - openjdk7
+  - openjdk8
```

Comment:
The most recent ClojureScript seemed to need Java 8 to work in Travis where the previous didn't, since the recent-clj build failed with openjdk7 and the main build succeeded.  I haven't yet investigated what changed.

Reply:
It may just be due to some dev time stuff in cljs. It requires a jdk7 minimum according to docs. However, for some extra jdk nashorn features it uses jdk8. I'm not sure the specifics.